great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -37,13 +46,18 @@
 from great_expectations.core.expectation_validation_result import get_metric_kwargs_id
 from great_expectations.core.id_dict import BatchKwargs
 from great_expectations.core.metric import ValidationMetricIdentifier
+from great_expectations.core.run_identifier import RunIdentifier
 from great_expectations.core.serializer import (
     AbstractConfigSerializer,
     DictConfigSerializer,
 )
+from great_expectations.core.usage_statistics.events import UsageStatsEvents
 from great_expectations.core.util import nested_update
 from great_expectations.core.yaml_handler import YAMLHandler
 from great_expectations.data_asset import DataAsset
+from great_expectations.data_context.config_validator.yaml_config_validator import (
+    _YamlConfigValidator,
```

Comment:
Are we ok with importing methods marked private? Does the underscore just denote that this isn't part of the public API?

Reply:
Yup the purpose is to denote that it is not to be used externally - this is how @anthonyburdi implemented it and I think it works here. We should perhaps standardize moving forward though